src/server/models/Product.models.js

```diff
@@ -0,0 +1,44 @@
+import sequelize from '../database/connection'
+import {DataTypes} from 'sequelize'
+//import User from './User.models'
+
+
+
+    const Product = sequelize.define('Product', {
+        // Model attributes are defined here
+        id: {
+          type: DataTypes.INTEGER,
+          autoIncrement: true,
+          primaryKey: true
+        },
+        title: {
+          type: DataTypes.STRING        
+        },
+        price: {
+          type: DataTypes.FLOAT        
+        // },
+        // user_id: {
+        //   type: DataTypes.INTEGER,
+        //   references: {
+        //     // This is a reference to another model
+        //     model: User,      
```

Comment:
You can uncomment this field if you move `hasMany` and `belongsTo` declarations to this file. In this case, there will be no cycling dependencies, since the Product model will not be imported into the User model.